pkg/operator/k8sutil/network.go

```diff
@@ -77,13 +79,13 @@ func ApplyMultus(net rookv1.NetworkSpec, objectMeta *metav1.ObjectMeta) error {
 			shortSyntax = true
 		}
 
-		isCsi := strings.Contains(objectMeta.Labels["app"], "csi-")
```

Comment:
Why the change to the csi pods? They no longer need the cluster network? Whey did we have the filter before?

Reply:
We had the filter to apply only the public network to the csi pods

Reply:
So now OSDs are the only ones that apply the cluster network, right? This means mon, mgr, and other daemons were applying both before, but will only apply the public network now.

Reply:
Yes, Correct